documentation/design-docs/dotnet-tools.md

```diff
@@ -663,6 +664,53 @@ Examples:
     Wrote 12576 bytes to file
     Complete
 
+REPORT
+
+    dotnet-gcdump report -f|--file <input_dump_path>
+    
+    Generate report into stdout from a previously generated gcdump or from a running process.
+    
+    Usage:
+      dotnet-gcdump report [options]
+    
+    Options:
+      -f, --file        The file to read gcdump from.
+      -p, --process-id  The process id to collect the trace.
+      -t, --report-type The type of report to generate. Available options: heapstat (default)
+
+Examples:
+
+    $ dotnet gcdump report -f 20200207_094403_19847.gcdump | head -9
+      5,080,860  GC Heap bytes
+         66,289  GC Heap objects
+
+    Object Bytes  Type
+        131,096  System.Byte[] (Bytes > 100K)  [System.Private.CoreLib.dll]
```

Comment:
Nit: There is a 1 space mis-alignment between the column headers and the rest of the columns. I assume these are intended to be aligned : )